src/local/butler/lint.py

```diff
@@ -187,22 +201,28 @@ def execute(_):
       f for f in file_paths
       if f.endswith('.py') and not is_auto_generated_file(f)
   ]
-  go_changed_file_paths = [f for f in file_paths if f.endswith('.go')]
-  yaml_changed_file_paths = [f for f in file_paths if f.endswith('.yaml')]
-
-  for file_path in py_changed_file_paths:
-    line_length_override = ''
-    if '_test.py' in file_path:
-      line_length_override = '--max-line-length=240'
-
-    # Use --score no to make output less noisy.
+  py_changed_test_file_paths, py_changed_nontest_file_paths = (
+      seperate_python_tests(py_changed_file_paths))
+
+  # Use --score no to make output less noisy.
+  base_pylint_cmd = 'pylint --score=no --jobs=0 '
+  if py_changed_nontest_file_paths:
+    _execute_command_and_track_error(base_pylint_cmd +
+                                     ' '.join(py_changed_nontest_file_paths))
```

Comment:
nit: Would it be better to use an f-string for consistency (e.g., lines 218&219-220)?
Same with line 213-214.

Reply:
Done.